client/src/pages/Signup.js

```diff
@@ -2,8 +2,8 @@ import React, { useState, } from 'react';
 import { CssBaseline, Typography, makeStyles, Container } from '@material-ui/core/';
 
 
-import SubmitButton from '../components/SubmitButton'
-import CustomTextField from '../components/CustomTextField'
+import SubmitButton from '../Components/SubmitButton'
+import CustomTextField from '../Components/CustomTextField'
```

Comment:
Please change them to lowercase. I am sorry again...

Reply:
done!